### ISSUE TITLE
Sails - Mainsail requires repair

### ISSUE THREAD

**Issue by smr547** [CanberraOceanRacingClub/namadgi3#794]
Ben from Hood Sails has advised that the mainsail is in excellent condition considering the miles that it has done since installation. However, he said it needs some repairs before N3 goes north for the Winter: 

* Repair leech line tension cord block on headboard and associated tears in sail
* Install dyneema cloth protection patches at high wear area

Ben has guaranteed a 24 hour turnaround time.

I recommend we remove the main and take it to Sydney as part of the working bee in March. We could combine this we servicing of the lifejackets -- see #524

**Comment by Christopher-w-green** [comment]
I recommend doing it as oart of the Sydney to Batemans Bay delivery

**Comment by pcbaston** [comment]
Where will we take the sail off for a day and then put it back? On the way
from Batemans to Port Stephens via Sydney? Is that in Chris' planning?
Boat isstill in Sydney with exchange now. What an opportunity!

Paul Baston

**Comment by smr547** [comment]
Indeed, an excellent opportunity if we could please have some volunteers to take the sail off for Hoods to pick up. @CORCpete and I have been on board for a week doing maintenance and have flights back to Canberra this afternoon. We are exhausted. I’ll refer to Committee for guidance. Thanks for your understanding and helpful suggestions!

**Comment by smr547** [comment]
I’ve just  asked Ben from hoods 
* Doeshe have staff to do the mainsail removal and reinstall next week?
* Can he do the repair next week if we remove and deliver

the answer was a firm no and no 

they are very busy

**Comment by pcbaston** [comment]
Steve,
If I was at home I'd come and do it, but unfortunately I'm in Bright and
Vic for three weeks. If I'd known you were doing maintenance, again, I
could have come to do my bit, but the bookings only shows you and Pete
having the boat. I assume it's cruising in Sydney.

I'll speak to Sam to see if there is a way to show maintenance windows
better,or were these just opportunities taken?

I was aware that Pete had arranged for job to be installed by Hoods during
his booking via GitHub comments.

Sorry, can't help for a while.
BB is a long haul too, but I'll see what I can do to contribute.

Paul Baston

**Comment by smr547** [comment]
Hi Paul,

Thank you for the email.

Pete's booking was for cruising but we did a lot of maintenance while on board culminating in the head sail change on Friday morning. The need to have the mainsail removed came as somewhat of an unwelcome surprise. Ben was prepared to take the No 4 for minor repairs (returning sail on Tuesday) but was not prepared to take the main -- I learnt later that Hoods are very busy next week and do not have time to do the main on a fast turnaround. In any event Dimitri had a booking starting same day. 

I posted the issue to GitHub with CC to Chris to ensure he got first wind of it and could consider the suggestion of a stopover in Sydney enroute to Port Stevens -- on first glance not an unreasonable suggestion given that the delivery is some weeks off. I regret not being able to call Chris to give him a heads up, but, as everyone should know by now, I use GitHub virtually as the sole mean of maintenance communication and, in any event, Peter and I were like blue-arsed flies and did not have time to chat. 

Chris is on a tight time line and not keen to adjust his schedule so we need to look for an alternative.  One suggested by Peter O is that we have the main shipped to Hoods during the maintenance week in Batemans Bay (18-22/3) -- or one of the volunteers could do the round trip (perhaps including other business they might have in Sydney). Namadgi's PFDs are in need of a service so perhaps we could ship them as well and do a job lot. 

I'll talk to MOSS about the PFDs and Peter O is going to talk to Hoods.

Thanks again for your offer to help. Your's a timely reminder that we need to get an APB out to the members about the maintenance working bee at Batemans -- whilst it's in the calendar it needs to be better advertised so volunteers can make plans.

I hope you are enjoying Bright, I believe it is lovely this time of year

kind regards
Steven

@Christopher-w-green (delivery skipper) @CORCpete @delcosta (for scheduling) @mgrybaitis (for info to Committee)

**Comment by Christopher-w-green** [comment]
It's not that I am refusing to adjust my plans. You make it sound like I'm
not being flexible. If you look a bit more carefully at the booking
calendar you'll see that there is a weekend in the middle of the proposed
passage to Port Stephens. Trying to get the sail to Hoods before the
weekend would leave two days to get from Batemans Bay up to Sydney on the
Friday to get the sail off and to the sail maker.  I assume that the
promised 24 hour turnaround (hyeah, right) does not include weekends, so
the earliest we'd see the sail back probably Tuesday, then refit the sail.
Heading off on the Wednesday gives one day to get to Port Stephens. And
that's being optimistic.

**Comment by smr547** [comment]
Dear Chris,

I'm sorry if I implied that you are inflexible -- It was not my intention and I certainly know that the opposite to be true. 

The fact is that I did not even look at the schedule for your delivery before posting the issue. I was simply putting up the first suggestion that came to mind that seemed half reasonable.  Looking more carefully at your schedule I agree that a stopover in Sydney is impractical unless the you are given more time for the delivery. That would seem impossible given the firm bookings either side -- not to mention regatta deadlines. 

The fact is that your schedule is tight given the limited time available -- I should have been more cognisant of  that before making the suggestion.

kind regards
Steven
